ui/app/AppLayouts/Chat/panels/communities/TokenHoldersPanel.qml

```diff
@@ -64,26 +61,87 @@ Control {
             placeholderText: enabled ? qsTr("Search") : qsTr("No placeholders to search")
         }
 
-        StatusListView {
-            id: holders
+        StatusBaseText {
+            Layout.fillWidth: true
 
+            visible: !root.preview
+            wrapMode: Text.Wrap
+            font.pixelSize: Style.current.primaryTextFontSize
+            color: Theme.palette.baseColor1
+            text: searcher.text.length > 0 ? qsTr("Search results") : qsTr("All %1 token holders").arg(root.tokenName)
+        }
+
+        StatusListView {
             Layout.fillWidth: true
             Layout.preferredHeight: childrenRect.height
-            leftMargin: -Style.current.padding
-            model: filteredModel
-            delegate: StatusListItem {
-                readonly property bool unknownHolder: model.ensName === ""
-                readonly property string formattedTitle: unknownHolder ? "?" : model.ensName
 
-                sensor.enabled: false
+            model: filteredModel
+            delegate: RowLayout {
                 width: ListView.view.width
-                title: formattedTitle
-                statusListItemTitle.visible: !unknownHolder
-                subTitle: model.walletAddress
-                asset.name: model.imageSource
-                asset.isImage: true
-                asset.isLetterIdenticon: unknownHolder
-                asset.color: Theme.palette.userCustomizationColors[d.red2Color]
+                spacing: Style.current.padding
+
+                StatusListItem {
+                    readonly property bool unknownHolder: model.ensName === ""
+                    readonly property string formattedTitle: unknownHolder ? "?" : model.ensName
+
+                    Layout.fillWidth: true
+
+                    leftPadding: 0
+                    rightPadding: 0
+                    sensor.enabled: false
+                    title: formattedTitle
+                    statusListItemTitle.visible: !unknownHolder
+                    subTitle: model.walletAddress
+                    asset.name: model.imageSource
+                    asset.isImage: true
+                    asset.isLetterIdenticon: unknownHolder
+                    asset.color: Theme.palette.userCustomizationColors[d.red2Color]
+                }
+
+                StatusComboBox {
+                    id: combo
+
+                    Layout.preferredWidth: 70
+                    Layout.preferredHeight: 44
+
+                    visible: root.isSelectorMode && amount > 1
+                    control.spacing: Style.current.halfPadding / 2
+                    model: amount
+                    size: StatusComboBox.Size.Small
```

Comment:
Yeah, I talked with `John` and he said it's not very likely to have more than 99 tokens, however, it could be. 
I will create a different task to change the width of the combo dynamically depending on the selected content so this pr can be integrated soon.